Dockerfile

```
@@ -9,11 +9,15 @@ FROM debian:buster AS build
 # Allow users to specify forked node, or specific commit
 # If not fallback to release branch (or alternatively just specify branch)
 ARG NODE_REPO=https://github.com/casper-network/casper-node.git
-ARG NODE_COMMIT=
 ARG CLIENT_REPO=https://github.com/casper-ecosystem/casper-client-rs.git
+ARG SIDECAR_REPO=https://github.com/casper-network/casper-sidecar.git
+
+ARG NODE_COMMIT=
+ARG NODE_GITBRANCH=release-2.0.0-rc3
+ARG CLIENT_GITBRANCH=feat-track-node-2.0
 ARG CLIENT_COMMIT=
```

Unfortunately, there is no branch/tag for RC3 release, and they broke `feat-track-node-2.0` 3 days ago - see https://github.com/casper-network/cctl/pull/34#issuecomment-2242748274.

For now I would recommend to lock particular commit for client:

```suggestion
ARG CLIENT_GITBRANCH=
ARG CLIENT_COMMIT=317051ddfb372d9404850c6a1787f225d6d32b4e
```

---

Fixed in 392d98c67f5617c1e6eada29416865984c12610d.